plugins/woocommerce/client/blocks/assets/js/blocks/product-reviews/template.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { InnerBlockTemplate } from '@wordpress/blocks';
+
+const TEMPLATE: InnerBlockTemplate[] = [
+	[ 'core/comments-title' ],
```

Comment:
As mentioned in the description, a lot of these blocks will become variations or custom blocks specifically for reviews.